rust_src/src/keymap.rs

```diff
@@ -5,13 +5,17 @@ use remacs_sys::{current_global_map as _current_global_map, globals, EmacsInt, L
                  CHAR_META};
 use remacs_sys::{Fcons, Fevent_convert_list, Ffset, Fmake_char_table, Fpurecopy, Fset};
 use remacs_sys::{Qautoload, Qkeymap, Qkeymapp, Qnil, Qt};
-use remacs_sys::{access_keymap, maybe_quit};
+use remacs_sys::{access_keymap, map_keymap_call, map_keymap_function_t, map_keymap_internal,
+                 maybe_quit};
 
 use data::{aref, indirect_function};
 use eval::autoload_do_load;
 use keyboard::lucid_event_type_list_p;
+use libc::c_void;
 use lisp::{defsubr, LispObject};
 use lists::nth;
+use obarray::intern;
+use std::ptr;
```

Comment:
`std` should be the first.

Reply:
So is the reason why it is first is because it is a Rust internal library?

This means, the arrangement should be:
Rust standard library > External crates > remacs_sys > everything else?

Reply:
correct. As documented in the contributing.

Reply:
I see.

```markdown
Imports are grouped with an empty line between groups. This lets rustfmt keep them ordered. The grouping is:

    Rust crates
    external crates
    remacs crates
    modules within the current crate
```

I've read this before but it didn't register with me at that time. Now I know. Thanks!